presentation.md

```diff
@@ -223,7 +223,78 @@ Below is the 3D diagram of Torus:
 
 # Week 5
 
-## 07 November 2022 to 11 November 2022
+## 7 November 2022 - 11 November 2022 (Nomnoml)
+
+---
+
+### ERD Diagram
+
+Below is the ERD diagram by using Nomnoml:
+
+![ERD diagram](ERD_nom.svg)
+
+---
+
+### Git Workflow
+
+Below is the flowchart of Git Workflow:
+
+![Flowchart of Git Workflow](Git_workflow.svg)
+
+---
+
+# Week 6
+
+## 14 November 2022 - 17 November 2022
+
+---
+
+### 14 November 2022
+
+| Command            | Description                    | Remarks |
+| ------------------ | ------------------------------ | ------- |
+| `cat /etc/passwd ` | User available                 |         |
+| `cat /etc/group`   | check user list available      |         |
+| `env`              | To print environment variables |         |
+| `apt-get update`   | To update apt packages.        |         |
+
+---
+
+### 15 November 2022
+
+| Command                    | Description                          | Remarks                             |
+| -------------------------- | ------------------------------------ | ----------------------------------- |
+| `apt-cache search docker ` | To search packages available in apt  |                                     |
+| `usermod -aG docker $USER` | To add user into docker group.       | docker run --rm -it wernight/funbox |
+| `ps -aux / tasklist`       | To check processors.                 |                                     |
+| `__ &`                     | To send the process into background. |                                     |
+
+---
+
+### 16 November 2022
+
+| Command      | Description                                                                             | Remarks                                                                  |
+| ------------ | --------------------------------------------------------------------------------------- | ------------------------------------------------------------------------ |
+| `-pm `       | Stand for parents, meaning that no error if existing, make parent directories as needed | m stand for mode, meaning set file mode (as in chmod), not a=rwx - umask |
```

Comment:
-pm is part of makedir command, it should be something like `makedir -pm ~/parentdir/childdir1/childdir2`